### ISSUE TITLE
Entering any kind of vehicle causes decrease in framerate that lasts until map reset

### ISSUE THREAD

**Issue by TinyDeskEngineer06** [Facepunch/garrysmod-issues#6006]
### Details
Entering vehicles causes a severe drop in FPS that persists until an admin cleanup is performed or the map is reloaded. During normal gameplay, with -noworkshop enabled, on the "None" branch, version 2024.08.09, I am able to reach an average of 150 FPS. Upon entering a vehicle, the FPS sharply drops to an average of 15. This occurs with any vehicle, on any map, and also affects the Dev and x86-64 branches, once again with -noworkshop enabled. Verifying the game files does not stop the issue.

### Steps to reproduce
1. Load any map
2. Spawn or find any vehicle
3. Enter vehicle

**Comment by robotboy655** [comment]
This appears to be an issue on your end. most likely GPU driver issue. Try updating or rolling back your GPU driver.

Either that, or its a non workshop addon you have installed.

**Comment by TinyDeskEngineer06** [comment]
I used +showbudget to try and profile the issue and it showed a large jump in the Sound category. The only non-workshop addon I can think of that's related to both players entering vehicles and sounds is... A script I made to play a sound once when a player enters a vehicle and nothing else. I disabled all my local addons and that finally fixed the issue, but I don't understand why it happened in the first place.
